ludwig/api.py

```diff
@@ -992,6 +1052,7 @@ def predict(
 
                     logger.info(f"Saved to: {output_directory}")
 
+            logger.info(f"Finished decoding in: {(time.time() - start_time):.2f}s.")
```

Comment:
Not totally sure we want to keep this one around since we're logging it in the generate call anyway? Unless you were using this to add visibility for `model.predict()` calls in which case I wouldn't set it to `Finished decoding in: ` since this is actually prediction and `decoding` is an implementation detail

Reply:
Makes sense. I changed this log to `Finished predicting in...`.

`generate()` and `predict()` are two separate functions, so that's why I added separate logging statements for each.